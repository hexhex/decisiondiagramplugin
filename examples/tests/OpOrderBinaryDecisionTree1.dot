digraph {
     b_1 -> leaf3 [label="else"];
     a -> b_1 [label="else"];
     b -> leaf2 [label="else"];
     b_1 -> leaf1_1 [label="b<x"];
     a -> b [label="a<x"];
     b -> leaf1 [label="b<x"];
     leaf1_1 [label="leaf1 [class1]"];
     leaf3 [label="leaf3 [class2]"];
     leaf2 [label="leaf2 [class2]"];
     leaf1 [label="leaf1 [class1]"];
     b_1 [label="b"];
     a [label="a"];
     b [label="b"];
}
