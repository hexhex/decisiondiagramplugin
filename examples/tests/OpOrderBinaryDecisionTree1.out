{elseedge(b_1,leaf2), elseedge(a,b), elseedge(b,leaf3), conditionaledge(b_1,leaf1_1,b,"<",x), conditionaledge(a,b_1,a,"<",x), conditionaledge(b,leaf1,b,"<",x), leafnode(leaf1_1,class1), leafnode(leaf3,class2), leafnode(leaf2,class2), leafnode(leaf1,class1), innernode(b_1), innernode(a), innernode(b), root(b)}
